packages/components/src/components/ScalarToggle/ScalarToggle.vue

```diff
@@ -19,7 +19,7 @@ function toggle() {
 }
 
 const variants = cva({
-  base: 'relative h-3.5 w-6 cursor-pointer rounded-full bg-b-3 transition-colors duration-300',
+  base: 'relative h-3.5 min-w-6 w-6 cursor-pointer rounded-full bg-b-3 transition-colors duration-300',
```

Comment:
Not sure this is what you're trying to fix but sometimes `shrink-0` is also what you want

Reply:
yees very true!